docs/source/install.rst

```diff
@@ -56,10 +77,12 @@ The following instructions assume you'll be using ucx-py on a CUDA enabled syste
     ../contrib/configure-devel --prefix=$CONDA_PREFIX --with-cuda=$CUDA_HOME --enable-mt CPPFLAGS="-I/$CUDA_HOME/include"
     make -j install
 
-3) Install UCX-PY
+UCX-PY
```

Comment:
This is just a question (or nitpicking, depending how you look at it), but what's the right way of writing this name? I've been using `UCX-Py`, is `UCX-PY` the right one?

Reply:
Fair question. Not sure. Happy to change to `UCX-Py` if that makes more sense.

Reply:
I don't mind it either way, just asking because I'd like to know for the long-term. The reason I've been writing `UCX-Py` is to projects like NumPy and CuPy (and yes, these are the correct ways to write them).

Reply:
Yeah I went ahead with your suggestion. Agree it seems reasonable 👍